core/src/components/segment/segment.tsx

```diff
@@ -181,11 +177,21 @@ export class Segment implements ComponentInterface {
 
     const value = this.valueAfterGesture;
     if (value !== undefined) {
-      this.ionChange.emit({ value });
```

Comment:
If you compare the changes side-by-side, you might think `ionChange` is no longer emitted here. While I did remove this event emission, the earlier function calls execute `setNextIndex` → `checkButton` → `ionChange.emit`.

It was removed at this point as it was redundant and causing an extra event emission (once `ionChange.emit` was introduced to `checkButton`).

Reply:
I'm not seeing `ionChange` fire when dragging the segment (on Windows Chrome), only when clicking.